django_cloud_tasks/models.py

```diff
@@ -0,0 +1,118 @@
+from typing import Optional
+from django.db import transaction
+from django.apps import apps
+
+from datetime import datetime
+from django.db import models
+from django_cloud_tasks import tasks
+
+class Pipeline(models.Model):
+    name = models.CharField(max_length=50)
+
+    def start(self):
+        routines = self.routines.filter(required_routine_vertices__id__isnull=True)
```

Comment:
Can be just `self.routines.filter(dependencies__id=None)`.
That's the magic of the `ManyToManyField` :)